src/main/webapp/dataset.xhtml

```diff
@@ -1251,13 +1253,18 @@
                                         <p:ajax process="@this" event="itemUnselect" />
                                     </p:autoComplete>
                                     <p:message for="dataverseLinkName"/>
+                                     </p:fragment>  
                                 </div>
                             </div>
                         </div>
                         <div class="button-block">
-                            <p:commandLink type="button" styleClass="btn btn-default"  update=":messagePanel,linkDatasetForm"
-                                           value="#{bundle['dataset.link.save']}" action="#{DatasetPage.saveLinkingDataverses()}"/>
-                            <button type="button" class="btn btn-default" onclick="PF('linkDatasetForm').hide();PF('blockDatasetForm').hide();">
+                            <p:commandLink id="saveLinkButton" type="button" styleClass="btn btn-default"   actionListener="#{DatasetPage.saveLinkingDataverses}" 
+                                           update=":messagePanel  linkNameContent @([id$=Messages])" 
```

Comment:
With the inclusion of `:messagePanel` in this update attribute, the "Validation Error" alert msg is displayed on the page behind the popup. I suggest to remove that, leaving just the "You must select a linking dataverse." msg under the autocomplete component in the popup for validation errors.

Reply:
I added a remoteCommand for success that will only update the messagePanel.

Reply:
Looks good. Works as expected.